### PR TITLE
Clean up buildrequest messages

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: master/buildbot/data/buildrequests.py [hunk 1]
```diff
@@ -164,9 +164,9 @@ class EntityType(types.Entity):
 
     @defer.inlineCallbacks
     def generateEvent(self, brids, event):
-        for _id in brids:
+        for id in brids:
```
Comment (tardyp) [inline master/buildbot/data/buildrequests.py]: even if this work, I'm not so found of using reserved words as identifiers. This is why I used _id, if you dont like the _ we can call it brid

**Comment by tardyp** [comment]
W: 20, 0: Unused import ascii2unicode (unused-import)

LGTM apart from that

File: master/buildbot/data/buildrequests.py (hunk 1 above)
Comment (djmitche) [inline master/buildbot/data/buildrequests.py]: oh, good point - I'll adjust

**Comment by jaredgrubb** [comment]
LGTM!

**Comment by sa2ajj** [comment]
(all green)

**Pull request merged**